components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -643,8 +643,6 @@ public static class OIDCConfigProperties {
         public static final String IS_SUBJECT_TOKEN_ENABLED = "isSubjectTokenEnabled";
         public static final String SUBJECT_TOKEN_EXPIRY_TIME = "subjectTokenExpiryTime";
         public static final int SUBJECT_TOKEN_EXPIRY_TIME_VALUE = 180;
-        public static final String IS_ACCESS_TOKEN_CLAIMS_SEPARATION_ENABLED =
```

Comment:
Is this added recently? Are we using a different constant for the server configuration?

Reply:
This is the sp oidc property name.

`OAuth.OpenIDConnect.EnableClaimsSeparationForAccessToken` is the server level config name.